src/components/AstronautsPage/components/AstronautsList/AstronautsList.tsx

```diff
@@ -1,4 +1,26 @@
 import React from 'react';
+import { Grid } from '@mui/material';
 import { AstronautsItem } from './components/AstronautsItem';
+import { selectAstronauts } from '../../selectors/astronautsSelectors';
+import type { Astronauts } from '../../types/Astronauts';
 
-export const AstronautsList = () => <AstronautsItem />;
+export const AstronautsList = () => {
+  const { astronauts } = selectAstronauts();
+
+  return (
+    <Grid container>
+      {astronauts.map(
+        ({ name, agency, profile_image }: Astronauts, index: number) => (
+          <Grid item xs={12} sm={6} md={4} lg={3} key={index}>
+            <AstronautsItem
+              key={index}
+              name={name}
+              agency={agency}
+              profile_image={profile_image}
+            />
+          </Grid>
+        ),
+      )}
+    </Grid>
```

Comment:
Please add logic whether data is loading or we don't have astronauts at all.

Reply:
For error, because backend can return zero astronauts, but it isn’t error

Reply:
So i wrote the rigth condition

Reply:
Nope, for error, it is a wrong condition. It is a two different logic. But error handling will be added in the next story, can be skipped for now